app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -14,11 +14,11 @@ class SupplementalApplicationPage extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      persistedApplication: cloneDeep(props.application),
-      application: props.application
+      persistedApplication: cloneDeep(props.application) // salefroce current perssited data, frozen copy. latest saved copy.
```

Comment:
A couple of typos in salesforce and persisted. If we're going to commit a comment I think we want it to be a complete sentence, maybe:

"A frozen copy of the application state that is currently persisted to salesforce. This is the latest saved copy.